testing/cli/test_cli_sct_register_to_template.py

```diff
@@ -28,6 +29,24 @@ def template_lpi(tmp_path_factory):
     return path_out
 
 
+@pytest.fixture(scope="module")
+def labels_discs(tmp_path_factory):
+    """Create a disc labels file."""
+    file_out = os.path.join(str(tmp_path_factory.mktemp("tmp_data")), 'labels_discs.nii.gz')
+    im_labels = Image(sct_test_path('t2', 'labels.nii.gz'))
+    im_labels.data = np.zeros(im_labels.data.shape)
+    # im_labels.data[23,53,26] = 3
+    # im_labels.data[24,24,26] = 4
+    # im_labels.data[25,17,26] = 5
+    # im_labels.data[26,1,26] = 6
+    im_labels.data[30,53,26] = 3
+    im_labels.data[31,34,26] = 4
+    im_labels.data[31,17,26] = 5
+    im_labels.data[32,1,26] = 6
```

Comment:
Since the old label coordinates were wrong, is there any reason to keep them? :stuck_out_tongue_closed_eyes: 

```suggestion
    im_labels.data[30,53,26] = 3
    im_labels.data[31,34,26] = 4
    im_labels.data[31,17,26] = 5
    im_labels.data[32,1,26] = 6
```

Reply:
> Since the old label coordinates were wrong, is there any reason to keep them? 😝

no reason, sorry for not cleaning up 😅 
do you mind removing it since you,re at it? thanks!